src/Stuart/City.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace Stuart;
+
+class City
```

Comment:
Could be nice you complete this enum helpers because i did not find the full list of cities

Reply:
Will have a look right now. But actually I don't think we should hardcode this anymore... we are opening new cities every week, it makes no sense to create a new release each time. Will push on your repository.